### DISCUSSION TITLE
Font size in ui.table

### DISCUSSION THREAD

**Discussion by mrhunsaker** [zauberzeug/nicegui#2260] [Q&A]
### Question

I am failing to find a way using CSS or adding slots to ui.table to get the font size of a table to be larger (e.g., 1.25rem)

**Comment by falkoschindler** [comment]
Here are two possible solutions.

1. Custom cell slot with CSS style definition:
    ```py
    columns = [{'name': 'name', 'label': 'Name', 'field': 'name'},]
    rows = [{'name': 'Alice'}, {'name': 'Bob'}, {'name': 'Carol'}]
    table = ui.table(columns=columns, rows=rows, row_key='name')
    table.add_slot('body-cell', '<td style="font-size: 1.25em">{{ props.row.name }}</td>')
    ```

2. CSS definitions in head HTML addressing a custom table class:
    ```py
    columns = [{'name': 'name', 'label': 'Name', 'field': 'name'},]
    rows = [{'name': 'Alice'}, {'name': 'Bob'}, {'name': 'Carol'}]
    ui.add_head_html('<style>.my-table tbody td { font-size: 1.25em }</style>')
    ui.table(columns=columns, rows=rows, row_key='name').classes('my-table')
    ```

**Comment by stepheweffie** [comment]
Query it. 
```py
ui.query('td').style('font-size: 1.25em;')
```

Or you can style the container holding the table after the query but you'll need both for this to work. Maybe it's a bit hackish but... 

```py
with ui.row().style('font-size: 1.25em;'):
    columns = [ data ]
    rows = [data]
    with ui.table():
```

Individual elements and labels have to be styled separately. 

The row style will override the query although it won't work alone. 

`with ui.row().style('font-size: 2.25em;'):`

<img width="1440" alt="Screen Shot 2023-12-29 at 4 08 17 PM" src="https://github.com/zauberzeug/nicegui/assets/1074503/fc1d9c61-10a0-4ede-9119-b3e05fbac736">

Individual elements in the table, e.g. the search icon and ui.select.value are done separately.
